plugins/omprog/omprog.c

```diff
@@ -398,6 +394,14 @@ cleanup(wrkrInstanceData_t *pWrkrData)
 	RETiRet;
 }
 
+BEGINfreeWrkrInstance
+CODESTARTfreeWrkrInstance
+	if (pWrkrData->bIsRunning) {
+		kill(pWrkrData->pid, SIGTERM);
```

Comment:
I would go for ```signalOnClose``` or maybe even ```signalTERMOnClose```. ```terminateOn...``` IMHO has a wrong co-notation of being somewhat required, at least for my mind ;-)